pkg/api/rest/configure_kubernikus.go

```diff
@@ -25,8 +18,7 @@ func configureFlags(api *operations.KubernikusAPI) {
 }
 
 func configureAPI(api *operations.KubernikusAPI) http.Handler {
-
-	return setupGlobalMiddleware(api.Serve(setupMiddlewares))
+	return api.Serve(setupMiddlewares)
```

Comment:
Can we change this to `return api.Serve` and remove the `setupMiddleware` function?
I find these two levels of middleware confusing and never quite understood the intended use case for the *inner* middleware chain.